users/templates/base.html

```diff
@@ -39,98 +40,64 @@
             </li>
             {% if user.user_type == admin %}
             <li>
-                {% if request.path|startswith:custom_users_list_url or request.path|startswith:'/employees/reports/author/' %}
-                <a class="active" href="{{ custom_users_list_url }}">
-                    <img src="{% static 'users/images/white/iko-03.png' %}">
-                    <span>{% trans 'Employees' %}</span>
-                </a>
-                {% else %}
-                <a href="{{ custom_users_list_url }}">
-                    <img src="{% static 'users/images/grey/ico-03.png' %}">
-                    <span>{% trans 'Employees' %}</span>
+                <a href="{{ custom_users_list_url }}"
+                   {% if request.path|startswith:custom_users_list_url or request.path|startswith:'/employees/reports/author/' %}
+                    class="active" {% endif %}>
+                    <span class="link-icon"><i class="fa fa-users"></i></span>
+                    <span class="link-text">{% trans 'Employees' %}</span>
                 </a>
-                {% endif %}
             </li>
             {% endif %}
             {% if user.user_type == manager or user.user_type == admin %}
             <li>
-                {% if request.path|startswith:custom_users_notifications %}
-                <a class="active" href="{{ custom_users_notifications }}">
-                    <img src="{% static 'users/images/white/notifications.png' %}">
-                    <span>{% trans 'Notifications' %}</span>
+                <a href="{{ custom_users_notifications }}" {% if request.path|startswith:custom_users_notifications %}
+                   class="active"{% endif %}>
+                    <span class="link-icon"><i class="fa fa-bell"></i></span>
+                    <span class="link-text">{% trans 'Notifications' %}</span>
                 </a>
-                {% else %}
-                <a href="{{ custom_users_notifications }}">
-                    <img src="{% static 'users/images/grey/notifications.png' %}">
-                    <span>{% trans 'Notifications' %}</span>
-                </a>
-                {% endif %}
             </li>
             <li>
-                {% if request.path|startswith:custom_projects_list_url or request.path|startswith:'/employees/reports/project/' %}
-                <a class="active" href="{{ custom_projects_list_url }}">
-                    <img src="{% static 'users/images/white/iko-04.png' %}">
-                    <span>{% trans 'Projects' %}</span>
-                </a>
-                {% else %}
-                <a href="{{ custom_projects_list_url }}">
-                    <img src="{% static 'users/images/grey/ico-04.png' %}">
-                    <span>{% trans 'Projects' %}</span>
+                <a href="{{ custom_projects_list_url }}"
+                   {% if request.path|startswith:custom_projects_list_url or request.path|startswith:'/employees/reports/project/' %}
```

Comment:
A few days ago we discussed with @kbeker that in `html` we should keep part of tag in one line.

Reply:
Done